scripts/reglock.py

```diff
@@ -57,7 +58,8 @@ def main():
         sys.exit("error: requested size not aligned to 1k")
     size = size // 1024
     if size > SIZE_MAX_KB:
-        sys.exit("error: requested size too big")
+        warnings.warn("warning: requested size too big; Setting to allowed maximum")
```

Comment:
A little bit more verbose: Requested size x larger than allowed y.
Maybe we should fail this case?

Reply:
@de-nordic I asked to allow 32 kB, which will decreased to 31 kB in immutable boot region setup. Workaround for PM  limitation.
That is what @michalek-no did.